ui/fuzzy-search-modal.go

```diff
@@ -99,13 +98,35 @@ func (fs *FuzzySearchModal) InitList(rooms map[id.RoomID]*RoomView) {
 		fs.roomList = append(fs.roomList, room.Room)
 		fs.roomTitles = append(fs.roomTitles, room.Room.GetTitle())
 	}
+	fs.changeHandler("") // initial load of unread rooms
 }
 
 func (fs *FuzzySearchModal) changeHandler(str string) {
 	// Get matches and display in result box
-	fs.matches = fuzzy.RankFindFold(str, fs.roomTitles)
-	if len(str) > 0 && len(fs.matches) > 0 {
+	if len(str) > 0 {
+		fs.matches = fuzzy.RankFindFold(str, fs.roomTitles)
 		sort.Sort(fs.matches)
+	} else {
+		// Set matches to unread rooms if no query
+		fs.matches = []fuzzy.Rank{}
+		for i, room := range fs.roomList {
+			if len(room.UnreadMessages) > 0 {
+				if room.UnreadCount() > 0 {
+					// Higher importance to rooms where notifications are not disabled
+					fs.matches = append(
```

Comment:
I am not entirely sure if this is the best approach. Let me know if I should maybe switch out the the type of `fs.matches` to something else.